Dockerfile

```diff
@@ -1,13 +1,17 @@
 FROM alpine
 
-RUN apk --update add redis
-RUN apk add --no-cache python3 py3-pip
 RUN apk add --no-cache git make musl-dev go
+RUN apk add --update nodejs npm
```

Comment:
we need to add docker file in a separate dir as it not the dockerfile of gridify